src/save_arrays.c

```diff
@@ -262,12 +262,12 @@ const struct SaveChunkHeader gSaveChunkHeaders[] = {
 };
 const int gNumSaveChunkHeaders = NELEMS(gSaveChunkHeaders);
 
-struct UnkStruct_0202E474 *sub_020270C4(SaveData *saveData) {
+struct UnkStruct_0202E474 *sub_020270C4(SaveData *saveData) { //Save_SafariZone_Get? conflicts with other one
```

Comment:
Does this mean you have enough info to name UnkStruct_0202E474 ?

Reply:
UnkStruct_0202E474 *should* be SafariZone, but we already  have one, so I'm at a bit of a loss here